CHANGELOG.md

```diff
@@ -7,7 +7,8 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Enhancements
 
-* None.
+* Add support for pinning SSL certificates used for https and realms
```

Comment:
Are docs updated to reflect this? (or a PR pending)

Reply:
Docs PR: https://github.com/realm/realm.io/pull/4569